osc-ui/src/main/java/org/osc/core/broker/view/PluginView.java

```diff
@@ -37,23 +33,18 @@
 import com.vaadin.ui.Panel;
 import com.vaadin.ui.TabSheet;
 import com.vaadin.ui.VerticalLayout;
-import com.vaadin.ui.themes.ValoTheme;
 
 @SuppressWarnings("serial")
 @Component(service={PluginView.class}, scope=ServiceScope.PROTOTYPE)
 public class PluginView extends VerticalLayout implements View {
 
-    private static final String MAINTENANCE_MANAGER_PLUGIN_GUID = "GUID-07FFF1BC-EA9E-426B-A247-4AF6BD12B350.html";
     private static final String MAINTENANCE_CONTROLLER_PLUGIN_GUID = "GUID-DD7DFB29-CB4F-4DD1-BFF5-21694F740D0E.html";
```

Comment:
I believe this is used to create the link to the help `?` button shown in the view bar at the top. It looks like currently (prior to these changes) the link is broke. Instead please use this link GUID-65309889-62B2-43BE-81CE-6A4B650AAFEE.html (links to https://10.3.205.173/WebHelp/GUID-65309889-62B2-43BE-81CE-6A4B650AAFEE.html) and rename the variable name MAINTENANCE_CONTROLLER_PLUGIN_GUID  -> HELP_PLUGIN_GUID